native/src/main/resources/csv_error.properties

```
@@ -0,0 +1,115 @@
+
+error.union.types.not.allowed.as.expected.type=\
+  union types are not allowed in the expected type, found ''{0}''
+
+error.invalid.array.member.in.expected.type=\
+  invalid type ''{0}'' in the expected array type
+
+error.cannot.found.field.in.csv=\
+  no matching header value is found for the required field ''{0}''
+
+error.csv.value.cannot.cast.into.expected.type=\
+  value ''{0}'' cannot be cast into ''{1}''
+
+error.invalid.expected.type=\
+  expected a array type, found ''{0}'' 
+
+error.invalid.token.while.reading.the.csv.data=\
+  error while reading the csv data ''{0}'' in line: ''{1}'', column: ''{2}''
+
+error.invalid.csv.data.format=\
+  invalid csv data format
+
+error.invalid.expected.array.size=\
+  invalid array size for expected array type, cannot be greater than ''{0}''
+
+error.invalid.expected.tuple.size=\
+  invalid array size for expected tuple type, cannot be greater than ''{0}''
+
+error.invalid.skip.column.query=\
+  invalid query found for skip column field, ''{0}''
+
+error.invalid.type.for.field=\
+  no mapping type found for value ''{0}'' in key ''{1}''
+
+error.invalid.type.for.array=\
+  value ''{0}'' in index ''{1}'' is not compatible with array type ''{2}''
+
+error.expected.type.can.only.contains.basic.types=\
+  expected type cannot contains types other than basic types, ''{0}'' not allowed
+
+error.invalid.conversion.for.array.to.map=\
+  value ''{0}'' cannot be cast into ''{1}'', because fields in ''{1}'' or the provided \
+  expected headers are not matching with the ''{0}''
+
+error.invalid.configurations=\
+  invalid configurations: ''{0}''
+
+error.invalid.format.for.skiplines=\
+  Invalid format for the skipLines field. Expected format: 'start-end'
+
+error.invalid.range.for.skiplines=\
+  Invalid range for the skipLines field. Start value must be less than or equal to end value.
+
+error.invalid.value.for.skiplines=\
+  Invalid input for the skipLines field. Both start and end values must be integers.
+
+error.inconsistent.header=\
+  Header ''{0}'' cannot be find in data rows
+
+error.invalid.custom.header.length=\
+  Invalid number of headers
+
+error.invalid.header.names.length=\
+  Invalid number of headers
+
+error.header.cannot.be.empty=\
+  The provided header row is empty
+
+error.no.field.for.header=\
+  No mapping field in the expected type for header ''{0}''
+
+error.duplicate.field=\
+  Duplicate field found in record fields: ''{0}''
+
+error.cannot.convert.into.exptype=\
+  The source value cannot convert in to the ''{0}''
+
+error.no.custom.header.provided=\
+  Custom headers should be provided
+
+error.headers.with.varying.length.not.supported=\
+  CSV data rows with varying headers are not yet supported
+
+error.header.value.cannot.be.empty=\
+  Header cannot be empty
+
+error.duplicate.header=\
+  Duplicate header found: ''{0}''
+
+error.invalid.union.conversion=\
+  The CSV cannot be converted into any of the uniform union types in ''{0}''
```

```suggestion
  The CSV value cannot be converted into any of the uniform union types in ''{0}''
```
Not sure whether this explanation is correct